frontend/src/app/utils/testResults.ts

```diff
@@ -0,0 +1,55 @@
+function getTestResult(result: MultiplexResult): TestResult {
```

Comment:
I've created another file named testResult 😅 please let me know if there is a better filename 😆